py_wtf/cli/__init__.py

```diff
@@ -107,10 +113,26 @@ def pick_project_dir(directory: Path) -> Path:
     return project_directory
 
 
+def parse_deps(maybe_deps: None | Sequence[str]) -> Sequence[str]:
+    if not maybe_deps:
+        return ()
+    return tuple(dep.split(" ")[0] for dep in maybe_deps if "extra" not in dep)
```

Comment:
I guess at some point we may want to expose information about conditional deps, but also we're not reimplementing pypi.org, so.

Reply:
I'm sure there's a better way to parse deps though :D